src/hooks/api/gallery/useGetGalleryList.tsx

```diff
@@ -1,39 +1,33 @@
 import { useQuery, type UseQueryOptions } from '@tanstack/react-query';
 
 import { get } from '~/libs/api';
-import { type SurveyType, type TargetType } from '~/remotes/gallery';
+import { type GalleryType } from '~/remotes/gallery';
 
 /**
- * @param gallery_id 마지막으로 조회된 갤러리의 id
+ * @param page  조회할 페이지를 입력
  * @param position 조회할 갤러리의 position을 입력 (ex. 'all')
  * @param order_type 정렬 기준을 입력
  * @param count 조회할 갤러리의 개수
  */
 interface Request {
-  gallery_id?: string;
+  page?: number;
   position?: string;
   order_type?: string;
   count?: number;
 }
 
-export interface GalleryType {
-  gallery_id: string;
-  target: TargetType;
-  survey: SurveyType;
-}
-
 interface Response {
-  total_page: string;
+  total_page: number;
   galleries: GalleryType[];
 }
 
 const useGetGalleryList = (request: Request, options?: UseQueryOptions<Response>) => {
   return useQuery<Response>({
     queryKey: ['gallerys', request],
     queryFn: () =>
-      get<Response>('/v1/gallerys', {
-        data: request,
-      }),
+      get<Response>(
+        `/v1/gallerys?page=${request.page}&job=${request.position}&order-type=${request.order_type}&count=${request.count}`,
+      ),
```

Comment:
```suggestion
      get<Response>(
        `/v1/gallerys`,
        {
         page=request.page,
         ...
        }
      ),
```

요롷게 쿼리 파라미터를 분리해주면 어떨까나용??

Reply:
오오 처음에 왜 안되지? 해서 저렇게 했는데, 다시 시도하니 잘 되네요!
그떄 뭔가 잘못 썼었나봐요 ㅜㅜ 감사합니다